Sources/web3swift/Utils/ENS/ENSResolver.swift

```diff
@@ -26,8 +26,7 @@ public extension ENS {
             case ABI
             case pubkey
             case text
-
-            func hash() -> String {
+            var hash: String {
```

Comment:
We have had a lot of breaking changes lately and I suggest starting to implement proper deprecation in future PRs.
Here is a small guide on how use `@available` annotations to deprecate API - https://www.hackingwithswift.com/example-code/language/how-to-use-available-to-deprecate-old-apis

Reply:
For sure @yaroslavyaroslav mentioned though that this will stop with `3.1.0`. It was just necessary due to certain issues that broke the lib anyway.

Reply:
@janndriessen I'm actually waiting those two last PRs from @pharms-eth to be merged to restrict any broken changes in future.